client/src/components/Sidebar/modules/MainMenu.tsx

```diff
@@ -202,21 +203,38 @@ export const Menu: React.FunctionComponent<MenuProps> = ({
         }
       >
         <button
-          className="sidebar-footer__account"
+          className="
+          sidebar-footer__account
+          w-bg-primary
+          w-text-white
+          w-flex
+          w-items-center
+          w-relative
+          w-p-0
+          w-w-full
+          w-appearance-none
+          w-border-0
+          w-overflow-hidden
+          md:w-px-5
+          md:w-py-3
+          hover:w-bg-primary
+          hover:w-text-white focus:w-bg-primary focus:w-text-white"
           title={strings.EDIT_YOUR_ACCOUNT}
           onClick={onClickAccountSettings}
           aria-label={strings.EDIT_YOUR_ACCOUNT}
           aria-haspopup="menu"
           aria-expanded={accountSettingsOpen ? 'true' : 'false'}
           type="button"
         >
-          <div className="avatar square avatar-on-dark">
+          <div className="avatar avatar-on-dark !w-w-[28px] !w-h-[28px]">
```

Comment:
I would have assumed we’d display avatars in consistent sizes elsewhere in the UI, so since there is no `avatar` component in templates / React currently, we should probably do this with a `--small` BEM modifier here?

Reply:
So there is apparently a .small class here https://github.com/wagtail/wagtail/blob/main/client/scss/objects/_avatar.scss#L27

However it's used elsewhere in the admin, so if I was to update it, it would break that other part of the admin (another argument for utility styling and tailwind). 

I'd vote to keep these arbitrary values here because this specific account icon is arbitrary sized itself, and only used here, so to me doesn't seem like much value to entangle the sizing here with more BEM. 

Let me know if I should update `.small` or maybe convert it to `--small` (and do a bunch of extra testing on the other spots its used)? 
or 
If it is okay as is?